src/birthder.js

```diff
@@ -68,6 +68,40 @@
     }
   }
 
+  /**
+   * Create a room for all users except `username`
```

Comment:
Create a channel and invite all the users to it except the one specified via `username`.